docs/tool/manifest.js

```diff
@@ -31,15 +31,27 @@ const packagePaths = glob( 'packages/*/package.json' )
  * @return {Array} Manifest
  */
 function getPackageManifest( packageFolderNames ) {
-	return packageFolderNames.map( ( folderName ) => {
+	return packageFolderNames.reduce( ( manifest, folderName ) => {
 		const path = `${ baseRepoUrl }/packages/${ folderName }/README.md`;
-		return {
+		const tocPath = `${ baseRepoUrl }/packages/${ folderName }/docs/toc.json`;
+
+		// First add any README files to the TOC
+		manifest.push( {
 			title: `@wordpress/${ folderName }`,
 			slug: `packages-${ folderName }`,
 			markdown_source: path,
 			parent: 'packages',
-		};
-	} );
+		} );
+
+		// Next add any items in the docs/toc.json if found.
+		if ( fs.existsSync( join( __dirname, '..', tocPath ) ) ) {
+			const toc = require( join( __dirname, '..', tocPath ) ).values();
+			for ( const item of toc ) {
```

Comment:
Would it work as:

```js
manifest.push( ...toc );
```

I checked https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push and I see it accepts also more than one element as arguments.

Reply:
Updated.